.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
This is slightly confusing, maybe?
Create a report for a problem you are encountering
or something like that.

Reply:
I agree. Thanks